src/Core/Domain/Product/QueryResult/ProductCustomizability.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\QueryResult;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\ProductCustomizabilitySettings;
+
+/**
+ * Transfers data of product customizability value
+ */
+class ProductCustomizability
```

Comment:
`ProductCustomizability` successfully indicate the topic but might be improved.
`ProductCustomizationCapabilities` ? `ProductCustomizationSettings` ?

Reply:
we have ProductCustomizationSettings at Domain\Product\ProductCustomizationSettings.php which holds related constants.
However the ProductCustomizability is a query result, so I wouldn't go for "...Settings", but I might call it Options or Capabilities as you suggested :thinking: